assets/prefabs/food/raw/CoconutMeat.prefab

```diff
@@ -0,0 +1,23 @@
+{
+    "parent": "engine:iconItem",
+    "Item": {
+        "icon": "Cooking:CoconutMeat",
+        "stackId": "Cooking:coconutmeat",
```

Comment:
Following conventions in food prefabs, this should be `"stackId": "Cooking:coconutMeat"`